package.json

```diff
@@ -1,10 +1,13 @@
 {
   "name": "webmaker.org",
   "version": "1.1.129",
+  "engines": {
+    "node": "0.10.x"
+  },
   "scripts": {
     "test": "grunt validate",
     "start": "grunt dev",
-    "postinstall": "bower install && grunt build",
+    "postinstall": "bower install && grunt build && npm run download-locales",
```

Comment:
Oh yea, I was gonna remove 'npm run download-locales' and the `engines` block as they are all for Heroku